.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: java
 
+sudo: true
```

Comment:
Can you change to `sudo: required` ? https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments